runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -21,64 +24,77 @@
 @Slf4j
 public class ManagementView extends AbstractView {
 
-    private final long layoutRefreshTimeout = 500;
+    static final Duration WORKFLOW_TIMEOUT = Duration.ofMinutes(10);
```

Comment:
these constants don't belong here. They actually look like they need to be part of the servercontext

Reply:
servercontext or runtime parameters ?

Reply:
probably from a ServerContext. Which means that WORKFLOW_TIMEOUT Duration should be provided to addNode as a parameter.=

Reply:
This is on the client side why would you want it to be part of the server context?

Reply:
As in a client should pass these constants when the methods are called, I'm guessing from an orchestrator the constants should come from the server context.

Reply:
Yes, but the orchestrator doesn't invoke this method. Its the "application" that calls this through the runtime.

Reply:
Then the application should provide these parameters

Reply:
Done.